exopy/tasks/tasks/instr_task.py

```diff
@@ -142,7 +142,7 @@ def test_driver(self):
             d_cls, starter = run_time[DRIVER_DEPENDENCY_ID][d_id]
             driver = starter.start(d_cls,
                                    profile['connections'][c_id],
-                                   profile['settings'][s_id])
+                                   profile['settings'].get(s_id, {}))
```

Comment:
Addressing the above command would make this unnecessary.

Reply:
I don't really understand what you mean. This change simply corrects an unnecessary difference with `start_driver`. It is actually needed for many instruments which don't have any settings.

Reply:
Sorry my bad I though for a second it handled a missing entry from the runtime...